client/react/components/SignUpBody/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Button } from 'semantic-ui-react';
+
+import SignUpForm from '../signUpForm';
+import './styles.css'
+
+export default ({
+    userType,
+    chooseUser,
+    onSubmit,
+    formValues,
+    isFormProcessing,
+    errorMessage
+}) => {
+    if (userType) {
+        const chooseUserButton = userType === process.env.USER_TYPE_TENANT ? (
+            <Button onClick={() => chooseUser(process.env.USER_TYPE_LANDLORD)}>Signup as a landlord</Button>
```

Comment:
We need a way to revert back to this state after selecting an option

Reply:
Handled in the sign up reducer